### PR TITLE
v1.1.1 tradução

### PR DESCRIPTION
**Pull request by denapo**
Brazilian Portuguese translation

### CODE REVIEW

File: translations/base-pt-BR.yaml [hunk 1]
```diff
@@ -0,0 +1,649 @@
...
+        xMinutesAgo: <x> minutos atrás
+        oneHourAgo: uma hora atrás
+        xHoursAgo: <x> horas atrás
+        oneDayAgo: um dia atrás
+        xDaysAgo: <x> dias atrás
+
+        # Short formats for times, e.g. '5h 23m'
+        secondsShort: <segundos>s
+        minutesAndSecondsShort: <minutos>m <segundos>s
+        hoursAndMinutesShort: <horas>h <minutos>s
```
Comment (Dimava) [inline translations/base-pt-BR.yaml:53]: Quoting from line 9:
```
# Placeholders:
#    Do *not* replace placeholders! Placeholders have a special syntax like
#    `Hotkey: <key>`. They are encapsulated within angle brackets. The correct
#    translation for this one in German for example would be: `Taste: <key>` (notice
#    how the placeholder stayed '<key>' and was not replaced!)```
Do not translate replacements, they are constant strings

File: translations/base-pt-BR.yaml [hunk 2]
```diff
@@ -0,0 +1,649 @@
...
+        belt: *esteira
+        splitter: *divisor
+        underground_belt: *underground_belt
+        miner: *mineradora
+        cutter: *corte
+        rotater: *rotação
+        stacker: *empilhador
+        mixer: *misturador
+        painter: *pintor
+        trash: *lixo
```
Comment (Dimava) [inline translations/base-pt-BR.yaml:617]: `&ptr` / `*ptr` in YAML as a pointer to another place, do not translate them
example:
```yaml
value: &myPointer 123
value2: *myPointer # will also be 123

```

**Comment by tobspr** [comment]
What @Dimava said.

File: translations/base-pt-BR.yaml [hunk 3]
```diff
@@ -48,11 +48,11 @@ global:
...
         # Short formats for times, e.g. '5h 23m'
-        secondsShort: <segundos>s
-        minutesAndSecondsShort: <minutos>m <segundos>s
-        hoursAndMinutesShort: <horas>h <minutos>s
+        secondsShort: <seconds>s
+        minutesAndSecondsShort: <minutes>m <seconds>s
+        hoursAndMinutesShort: <hours>h <minutes>s
 
-        xMinutes: <x> minutos
+        xMinutes: <x> minutes
```
Comment (Dimava) [inline translations/base-pt-BR.yaml:55]: and here "minutos" should be translated

**Comment by denapo** [comment]
all orders have been placed and are following the latest updates

**Comment by tobspr** [comment]
Thanks a lot!

**Pull request merged**